### PR TITLE
SDCB-5362 update query params and data definitions, create unit tests for base clases

### PR DESCRIPTION
**Pull request by piotrsapiejewskismartbear**

### CODE REVIEW

File: src/api/APIListServices.spec.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  });
+
+  describe('@active', () => {
+    it('should initialize proper endpoint path', () => {
+      const result = service.active();
+      expect(result.toUrl()).toEqual('/me/services/active');
+      expect(result).toBeInstanceOf(APIList);
+    });
+
+    it('should initialize proper endpoint path', () => {
```
Comment (marisuch) [inline src/api/APIListServices.spec.ts:40]: Duplicated test description

File: src/api/APIListFiles.spec.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    service = new APIListFiles(api);
+  });
+
+  it('should initialize proper endpoint path', () => {
+    expect(service.toUrl()).toEqual('/files');
+  });
+
+  describe('@upload', () => {
+    it('should throw if used in browser', () => {
```
Comment (marisuch) [inline src/api/APIListFiles.spec.ts:28]: throw error :)

File: src/api/APIListSmartbearTunnels.spec.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    service = new APIListSmartbearTunnels(parent);
+  });
+
+  it('should initialize proper endpoint path', () => {
+    expect(service.toUrl()).toEqual('/me/tunnels');
+  });
+
+  describe('@active', () => {
+    it('should initialize proper endpoint path', () => {
```
Comment (marisuch) [inline src/api/APIListSmartbearTunnels.spec.ts:27]: Duplicated test description

File: src/api/APIListServices.spec.ts (hunk 1 above)
Comment (piotrsapiejewskismartbear) [reply src/api/APIListServices.spec.ts]: fixed

File: src/api/APIListFiles.spec.ts (hunk 2 above)
Comment (piotrsapiejewskismartbear) [reply src/api/APIListFiles.spec.ts]: fixed

File: src/api/APIListSmartbearTunnels.spec.ts (hunk 3 above)
Comment (piotrsapiejewskismartbear) [reply src/api/APIListSmartbearTunnels.spec.ts]: fixed

**Review by marisuch — APPROVED** [review]

**Pull request merged**